evo_researcher/autonolas/research.py

```diff
@@ -905,9 +934,6 @@ def extract_and_sort_sentences(
     doc_question = nlp(event_question)
     event_date = extract_event_date(doc_question)
 
-    # Create embedding for event question with Spacy embedder model
-    query_emb = nlp(event_question)
```

Comment:
This is the same as `doc_question` above

Reply:
good catch!